src/common/transformations/tests/common_optimizations/group_normalization_fusion_tests.cpp

```diff
@@ -0,0 +1,570 @@
+                FAIL() << "Unexpected shape of instance norm beta - expected either empty shape (which means that it "
+                          "will not "
+                          "be put in the graph) or shape with exactly num_groups elements that can be merged with the "
+                          "result "
+                          "of MVN.";
+
+            if ((group_norm_gamma_shape != Shape{}) && (shape_size(group_norm_gamma_shape) != num_channels))
+                FAIL()
+                    << "Unexpected shape of group norm gamma - expected either empty shape (which means that it will "
+                       "not "
+                       "be put in the graph) or shape with exactly num_channels elements that can be merged with the "
+                       "result "
+                       "of instance norm.";
+
+            if ((group_norm_beta_shape != Shape{}) && (shape_size(group_norm_gamma_shape) != num_channels))
+                FAIL()
+                    << "Unexpected shape of group norm beta - expected either empty shape (which means that it will "
+                       "not "
+                       "be put in the graph) or shape with exactly num_channels elements that can be merged with the "
+                       "result "
+                       "of instance norm.";
+        }
+        auto instance_norm_gamma_present = (instance_norm_gamma_shape != Shape{});
+        auto instance_norm_beta_present = (instance_norm_beta_shape != Shape{});
+        auto group_norm_beta_present = (group_norm_beta_shape != Shape{});
+        auto group_norm_gamma_present = (group_norm_gamma_shape != Shape{});
+
+        if (positive_test) {
+            instance_norm_gamma_present =
+                instance_norm_gamma_present && (shape_size(instance_norm_gamma_shape) == num_groups);
+            instance_norm_beta_present =
+                instance_norm_beta_present && (shape_size(instance_norm_beta_shape) == num_groups);
+            group_norm_beta_present = group_norm_beta_present && (shape_size(group_norm_beta_shape) == num_channels);
+            group_norm_gamma_present = group_norm_gamma_present && (shape_size(group_norm_gamma_shape) == num_channels);
+        }
+
+        auto instance_norm_gamma_vals = std::vector<T_in_gamma_store_t>();
+        if (instance_norm_gamma_present)
+            instance_norm_gamma_vals =
+                test::utils::generateVector<T_in_gamma_elem_t>(shape_size(instance_norm_gamma_shape));
+
+        auto instance_norm_beta_vals = std::vector<T_in_beta_store_t>();
+        if (instance_norm_beta_present)
+            instance_norm_beta_vals =
+                test::utils::generateVector<T_in_beta_elem_t>(shape_size(instance_norm_beta_shape));
+
+        auto group_norm_gamma_vals = std::vector<T_gn_gamma_store_t>();
+        if (group_norm_gamma_present)
+            group_norm_gamma_vals = test::utils::generateVector<T_gn_gamma_elem_t>(shape_size(group_norm_gamma_shape));
```

Comment:
```suggestion
        const auto group_norm_gamma_vals = test::utils::generateVector<ElementType>(group_norm_gamma_present ? shape_size(group_norm_gamma_shape) :  0);
```
?
It may require update test utils `generateVector` to create empty vector for input size 0.

Reply:
This code has been refactored in new version, so it no longer applies